frame/contracts/fixtures/drain.wat

```diff
@@ -33,7 +33,9 @@
 			)
 		)
 
-		;; Self-destruct by sending full balance to the 0 address.
+		;; Try to self-destruct by sending full balance to the 0 address.
+		;; All the free balance will be send away but the contract's which is a valid
+		;; thing to do because the storage deposits will keep the account alive.
```

Comment:
```suggestion
		;; All the *free* balance will be send away, which is a valid thing to do
		;; because the storage deposits will keep the account alive.
```

I had to read this a couple times to get it.